plaso/parsers/presets.py

```diff
@@ -3,75 +3,105 @@
 
 from __future__ import unicode_literals
 
+import yaml
+
+from plaso.lib import errors
+
+
 class ParserPresetDefinition(object):
   """Parser and parser plugin preset definition.
 
   Attributes:
     name (str): name of the preset.
-    parsers (str): names of parser and parser plugins.
+    parsers (list[str]): names of parser and parser plugins.
   """
 
-  def __init__(self):
-    """Initializes a parser and parser plugin preset definition."""
+  def __init__(self, name, parsers):
+    """Initializes a parser and parser plugin preset definition.
+
+    Attributes:
+      name (str): name of the preset.
+      parsers (list[str]): names of parser and parser plugins.
+    """
     super(ParserPresetDefinition, self).__init__()
-    self.name = None
-    self.parsers = None
+    self.name = name
+    self.parsers = parsers
 
 
 class ParserPresets(object):
```

Comment:
Rename to PresetManager or PresetReader or similar. This class doesn't contain preset definitions like it used to.

Reply:
ack